.github/workflows/update-template.yml

```diff
@@ -92,9 +93,19 @@ jobs:
     runs-on: ubuntu-latest
     needs: [update-template]
     steps:
+      - uses: ben-z/actions-comment-on-issue@1.0.2
+        with:
+          message: ":white_check_mark: The template has been updated in Template Registry."
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: ben-z/actions-comment-on-issue@1.0.2
+        if: needs.update-template.outputs.status == 'InVerification'
```

Comment:
@AmyJZhao , please see my comment above about reusing the "needs-more-verification" workflow